firebase/functions/index.js

```diff
@@ -4,158 +4,132 @@
 
 const functions = require('firebase-functions');
 const admin = require('firebase-admin');
-const {PubSub} = require(`@google-cloud/pubsub`);
+const { PubSub } = require(`@google-cloud/pubsub`);
 const pubsub = new PubSub();
 
-const EXPIRY_MS = 1000 * 60 * 60 * 24;
-
 admin.initializeApp(functions.config().firebase);
 const db = admin.firestore();
 
-function deleteRun(port, port_doc, runid) {
-  const rundoc = port_doc.collection('runid').doc(runid);
-  rundoc.collection('test').get().then(function(snapshot) {
-    snapshot.forEach(function(run_test) {
-      console.log('Deleting', port, runid, run_test.id);
-      rundoc.collection('test').doc(run_test.id)
-        .delete().catch((error) => {
-          console.error('Error deleting', port, runid, run_test.id, error);
-        });
-    });
-  });
-  rundoc.delete().catch((error) => {
-    console.error('Error deleting', port, runid, error);
-  });
-}
-
 exports.daq_firestore = functions.pubsub.topic('daq_runner').onPublish((event) => {
   const message = event.json;
   const origin = event.attributes.origin;
-  const site_name = event.attributes.site_name || 'unknown';
```

Comment:
this refactoring (changing the names of things) needs to be done independently of any functionality change -- otherwise it's near impossible to do an effective code review.